libs/common-services/src/main/java/uk/gov/di/ipv/core/library/enums/Vot.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.core.library.enums;
+
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.gpg45.enums.Gpg45Profile;
+
+import java.util.List;
+
+@ExcludeFromGeneratedCoverageReport
+public enum Vot {
+    P0(List.of(), List.of()),
+    P2(List.of(Gpg45Profile.M1A, Gpg45Profile.M1B, Gpg45Profile.M2B), List.of()),
+    PCL250(List.of(), List.of(OperationalProfile.PCL250)),
+    PCL200(List.of(), List.of(OperationalProfile.PCL250, OperationalProfile.PCL200));
+
+    private final List<Gpg45Profile> validGpg45Profiles;
+    private final List<OperationalProfile> validOperationalProfiles;
+
+    Vot(List<Gpg45Profile> validGpg45Profiles, List<OperationalProfile> validOperationalProfiles) {
+        this.validGpg45Profiles = validGpg45Profiles;
+        this.validOperationalProfiles = validOperationalProfiles;
+    }
+
+    public List<Gpg45Profile> getValidGpg45Profiles() {
```

Comment:
Is this actually `getSupportedGpg45Profiles`?

Reply:
Ha. That's exacly what I started with and then switched to `valid` at the last minute. I think mostly because it was less verbose. Maybe that was a bad decision. I'll switch it back.